### ISSUE TITLE
no-unused-vars not working for params of empty functions

### ISSUE THREAD

**Issue by fvictorio** [protofire/solhint#136]
Code:
```solidity
contract Foo {
  function foo(uint x) public returns (uint) {
  }
}
```

Config:
```json
{
  "rules": {
    "no-unused-vars": "error"
  }
}
```

No error is emitted. Notice that this does emit:
```solidity
contract Foo {
  function foo(uint x) public returns (uint) {
    return 42;
  }
}
```

**Comment by fvictorio** [comment]
I declare this to be Expected Behavior

![crowning](https://user-images.githubusercontent.com/417134/81082517-c00efd80-8ec9-11ea-8f81-e99d0843a311.gif)
